CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 ## [Unreleased]
 
 ### Changed
+- Moved the bst package into logic. For developers: This requires the regeneration of antlr sources, execute: gradlew generateSource
```

Comment:
One could argue if this should be in the change log as it is mainly a code issue. However, as there are some earlier entries like that and there is the need of antlr source regeneration it might just as well stay there.